services/monitors/monitors/validation.go

```diff
@@ -10,7 +10,3 @@ var monitorExpressionValidators = Validators{
 		Or(Equals("comparison_operator", "EQUAL_TO"), Equals("comparison_operator", "NOT_EQUAL_TO")),
 	),
 }
-
-func (monitor *Monitor) Validate(obj map[string]any) error {
-	return monitorExpressionValidators.Check(obj["expression"].([]any)[0].(map[string]any))
```

Comment:
For Rule, we can keep the same validation here because it's possible to set tolerance for EQUAL_TO or NOT_EQUAL_TO comparison operator.
Just rename the method `monitorExpressionValidators` by `monitorRuleValidators`.